docs/model-monitoring/index.md

```diff
@@ -126,4 +127,15 @@ See an example of batch input in the [Serving pre-trained ML/DL models](../tutor
 ## Alerts and notifications
 
 You can set up {ref}`alerts` to inform you about suspected and detected issues in the model monitoring functions. 
-And you can use {ref}`notifications` to notify about the status of runs and pipelines.
\ No newline at end of file
+And you can use {ref}`notifications` to notify about the status of runs and pipelines.
+
+## Scale limitations
+
+When ramping up the scale of your model monitoring, take note of these limitations. Application results refers to both metrics and results.
+- Number of MEP application results within a project = MEP *  sum (MM applications' results within the project)  should be < 50K. For example, this can be 5000 MEPs with 10 results or 1000 MEPs with 50 results.
+- Every project with MM enabled takes at least 3 system pods: stream (might have replication), controller, writer. And a pod per application (including the histogram app). By default 4 workers per app (configurable, config applies to all apps).
+- Number of projects and number of MEP per project affects
```

Comment:
affects what?

Reply:
@theSaarco well - that is for you to fill in please